src/pages/html/elements/ol-tag/index.md

```diff
@@ -3,13 +3,26 @@ title: Ol Tag
 ---
 ## Ol Tag
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/html/elements/ol-tag/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+The `<ol>` tag is used to represent an ordered set of items and it typically renders each item with either a letter or number associated with it.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+A simple example of an ordered list:
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+```html
+<ol>
+  <li>First</li>
+  <li>Second</li>
+  <li>Third</li>
+</ol>
+```
+
+The above example would render the following in the browser:
+
+1. First
+2. Second
+3. Third
 
 #### More Information:
```

Comment:
You can remove lines 24-28, since you already started writing the article.

Reply:
here you go, @jamesjpk123 - thanks for the review again! d3192fd4f3a557ce7a624448b2b8d3d845626408